### ISSUE TITLE
[Feature] port contractURI when creating contract to bridge HRC721 tokens

### ISSUE THREAD

**Issue by Otium420** [harmony-one/harmony#3962]
# Problem/limitation At Hand

**TL;DR:** include `contractURI` in the contract that is created when bridging a HRC721 token to Ethereum the first time.

Harmony is one the probably one of the bests blockchains to launch NFT games of it.

It has low fees and the transaction finality is superb.

However, for game creators, moving outside of Ethereum is a risk, since the userbase is much smaller in any other blockchain, and the secondary marketplaces are nowhere near as popular as OpenSea. Nonetheless, staying on ethereum (with their gas fees) limits too much the possibility of any game..

My plan to overcome the above situation is to: **have a game native to Harmony, but allow the NFT tokens of the game to be easily bridged to for them to be sold**, the opposite operation will also be possible, allowing people to join the game (and Harmony) by buying wrapped ERC721 tokens on OpenSea and bridging to Harmony. 

Currently, I am facing a problem with the above plan. When trying to bridge a new HRC721 to Ethereum, I have no way to set the metadata needed in OpenSea for the secondary market because I do not own the contract that is used to create the "Wrapped" NFT on Ethereum. I cannot:
- Configure the wallet that will receive the secondary market fees
- Set the secondary market fee
- Configure imgs, description, etc.

Allowing me to **own** the contract used to wrap the tokens would be ideal, but may impact the **permissionless** aspect of the bridge. Instead of that, I propose the `contractURI` returned string from the original contracto to be included in the newly created wrapping contract, similarly to how is done with `baseURI`.

[**contractURI** allows to set the metadata required above for OpenSea, without requiring me to own the contract.](https://docs.opensea.io/docs/contract-level-metadata).

**P.D: This may also apply to HRC1155**

**Which component?**

- [ ] consensus
- [ ] deploy script
- [ ] wallet
- [ ] explorer
- [ ] node script
- [ ] test automation
- [ ] demo app
- [ ] others ...

# Proposed Solution

include `contractURI` in the contract that is created when bridging a HRC721 token to Ethereum the first time.

# Alternatives Considered

Allowing to use a "custom contract" when bridging an HRC721 token for their wrapped version on Ethereum

# Additional Context

I am building https://twitter.com/kurorobeast

We currently have 5k followers on twitter and 7k members on our limited entry discord

**Comment by Otium420** [comment]
Hey @jhd2best, Bruno Marshall mentioned you would be the best person to handle this.

**Comment by Severino** [comment]
The HRC721 bridge is still quite new, but I think this is a crucial point to maintain the value of bridged NFTs.

@gupadhyaya @LeoHChen @JackyWYX is this a missing feature of the bridge? Are there plans to include this?
